netbsd/NetBSDProcessList.c

```diff
@@ -49,23 +50,64 @@ static char const *freqSysctls[] = {
    NULL
 };
 
+static void NetBSDProcessList_updateCPUcount(ProcessList* super) {
+   NetBSDProcessList* opl = (NetBSDProcessList*) super;
+   const int nmib[] = { CTL_HW, HW_NCPU };
+   const int mib[] = { CTL_HW, HW_NCPUONLINE };
```

Comment:
Somewhat more telling variable names would be fine.

Reply:
Put a short comment on the macro and the reference to it from `sysctl.h`.